addons/point_of_sale/models/pos_order.py

```diff
@@ -237,6 +205,7 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
         states={'done': [('readonly', True)], 'invoiced': [('readonly', True)]},
     )
     amount_tax = fields.Float(string='Taxes', digits=0, readonly=True, required=True)
+    tax_details = fields.Json(string='Tax amount per tax id', readonly=True)
```

Comment:
Hello, this field is useless, it is just a copy of values of the order lines (moreover, storing for each order a string containing taxes data seems a really bad idea as storing relational data into a string will make inconsistencies).

Reply:
@pimodoo You don't have this information on order lines. The purpose of this field is to avoid inconsistencies between js-side and python-side. This will never be editable.